staging/src/k8s.io/apiserver/pkg/storage/tests/cacher_test.go

```diff
@@ -112,11 +122,18 @@ func newTestCacher(s storage.Interface, cap int) (*cacherstorage.Cacher, storage
 		NewFunc:        func() runtime.Object { return &example.Pod{} },
 		NewListFunc:    func() runtime.Object { return &example.PodList{} },
 		Codec:          codecs.LegacyCodec(examplev1.SchemeGroupVersion),
+		Clock:          clock,
 	}
 	cacher, err := cacherstorage.NewCacherFromConfig(config)
 	return cacher, v, err
 }
 
+func newTestCacherWithFakeClock(s storage.Interface, cap int) (*cacherstorage.Cacher, storage.Versioner, *clock.FakeClock, error) {
```

Comment:
Let's remove this function, it's cleaner if you create the clock yourself in the single test where you need it and simply pass it here.

Reply:
done, it's cleaner.